ioc_finder/ioc_grammars.py

```diff
@@ -107,14 +109,13 @@
     + Or([domain_name, "[" + ipv4_address + "]", "[IPv6:" + ipv6_address + "]"])("email_address_domain")
 )
 
-scheme_literals = (CaselessLiteral(i) for i in schemes)
-url_scheme = Or(scheme_literals)
-port = Combine(":" + Word(nums))
+url_scheme = one_of(schemes)
```

Comment:
Keep the caselessness:

```suggestion
url_scheme = one_of(schemes, caseless=True)
```